src/main/java/teammates/storage/sqlentity/Instructor.java

```diff
@@ -39,6 +39,15 @@ protected Instructor() {
         // required by Hibernate
     }
 
+    public Instructor(String registrationKey, boolean isDisplayedToStudents, String displayName,
+            InstructorPermissionRole role, String instructorPrivileges) {
+        this.registrationKey = registrationKey;
+        this.isDisplayedToStudents = isDisplayedToStudents;
+        this.displayName = displayName;
+        this.role = role;
+        this.instructorPrivileges = instructorPrivileges;
```

Comment:
Use the setter to the values are sanitised. Also missing attributes in user class.